lib/Doctrine/Inflector/Rules/English/Inflectible.php

```diff
@@ -173,12 +173,12 @@ public static function getIrregular(): iterable
         yield new Substitution(new Word('foe'), new Word('foes'));
         yield new Substitution(new Word('grave'), new Word('graves'));
         yield new Substitution(new Word('hoax'), new Word('hoaxes'));
-        yield new Substitution(new Word('medium'), new Word('media'));
```

Comment:
Should we sort the list, so as to avoid such duplicates?

Reply:
I was wondering as well ... I'll do it